.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 .DS_Store
 .env
 npm-debug.log
+bundle.js
```

Comment:
`bundle.js` is compiled into `build` directory and `build` dir is already gitignored; is that sufficient?